main.go

```diff
@@ -40,11 +47,16 @@ func main() {
 				if readErr == io.EOF {
 					break
 				}
+				count++
 				continue
-			} else if bytes.Compare(content,checkEquality) != 0 {
+			} else if bytes.Compare(content, checkEquality) != 0 {
 				if skip {
 					skip = false
+					if *countParameter {
+						fmt.Printf("      %v ", count)
```

Comment:
Why v?

Reply:
i dont know but it works

Reply:
You don't know? Figure it out.

Reply:
v = the value in a default format

Reply:
but i should use %d i guess?

Reply:
I didn't ask what it does, but why you chosen it.

Reply:
Exactly.